README.md

```diff
@@ -97,6 +97,40 @@ On IDE:
 
     * `mvn clean package -DskipTests` to just create the jar file
 
-## Deploying To Elastic Beanstalk
+## Deployment
+
+### Git Strategy
+
+`master` is stable but bleeding edge. Cut feature branches off of `master`.
+Send PRs to be merged into `master`.
+
+`master` ==gets merged to==> `qa` ==gets merged into==> `production`.
+
+The `qa` branch should be what's running in the QA environment.
+The `production` branch should be what's running in the production environment.
+
+### AWS Elastic Beanstalk
+1. `.ebextensions` directory needed at application's root directory
+2. `.ebextensions/environmentvariables.config` to store environment variables. For environment variables that needs to be hidden,
+3. `Procfile` to start Spring Boot app after deployment.
+4. `eb init -i --profile <<your AWS profile>>`
+5. Initial creation of instance on Beanstalk:
+
+Please use the instance profile of _cloudwatchable-beanstalk_.
```

Comment:
I would like to confirm that for AWS production instance, we are using instance profile and it is the instance profile **cloudwatchable-beanstalk**